root.html

```diff
@@ -57,7 +57,7 @@
         }
     </style>
 </head>
-<body class='font--open_sans'>
+<body class='font--open_sans enable-animations'>
```

Comment:
please set the name for the animation to be executed in case we add more in the future so it is explicit. Or is it just to enable them and there are going to be additional animation classes appended?